src/combobox/nls/ComboBox.ts

```diff
@@ -0,0 +1,9 @@
+const bundlePath = 'src/combobox/nls/common';
+const locales: string[] = [];
+
+const messages = {
+	clear: 'clear combo box',
```

Comment:
possibly a question for @smhigley . Do these need to explicitly say `combo box` or could they be moved to common and simply read: `clear`, `open`, `close` etc...? Would save repetition.